src/disqus.js

```diff
@@ -536,7 +536,7 @@ function DisqusJS(config) {
                     el.innerHTML = msg;
                     const aTag = el.getElementsByTagName('a');
                     for (const i of aTag) {
-                        const link = decodeURIComponent(i.href.replace(/https:\/\/disq.us\/url\?url=/g, '')).replace(/(.*):.+cuid=.*/, '$1');
+                        const link = decodeURIComponent(i.href.replaceAll('https://disq.us/url?url=', '')).replace(/(.*):.+cuid=.*/, '$1');
```

Comment:
`String#replaceAll` has bad browser compatiblity.

Reply:
I thought Babel can already transform it 😕 Will revert to `replace`.